### DISCUSSION TITLE
[Support]: Frigate deleting all recording "running storage maintenance" when over 10TB are still left free.

### DISCUSSION THREAD

**Discussion by psychopat81** [blakeblackshear/frigate#15471] [General Support]
### Describe the problem you are having

I have all my cameras setup for continuous recording: 

    record:
      enabled: true
      retain:
        days: 30
        mode: all
      events:
        retain:
          default: 90
          mode: motion

Every 10 mins Frigate is deleting all recordings and starting over. 
See the logs here: 

![frigate deleting recordings](https://github.com/user-attachments/assets/f362948f-2cb0-405d-becd-3ac64f7f4fb1)

There's plenty of space available on the drive. 

![space left](https://github.com/user-attachments/assets/6747daaa-41fb-42c5-9093-dbd367e9921e)

![storage](https://github.com/user-attachments/assets/54234af8-4a34-4f7d-96ef-76da1505160c)

### Version

0.14.1-f4f3cfa

### What browser(s) are you using?

Google Chrome 131.0.6778.108

### Frigate config file

```yaml
record:
      enabled: true
      retain:
        days: 30
        mode: all
      events:
        retain:
          default: 90
          mode: motion
```

### Relevant Frigate log output

```shell
info
2024-12-12 09:14:10
frigate.storage
Less than 1 hour of recording space left, running storage maintenance...
error
2024-12-12 09:14:10
frigate.storage
Could not clear 105669675.11000001 MB, currently 476.09999999999974 MB have been cleared. Retained recordings must be deleted.
```

### Relevant go2rtc log output

```shell
none
```

### FFprobe output from your camera

```shell
none
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
unraid
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Hikvision 52

**Comment by NickM-27** [comment]
looks like one of the cameras might be having problems saving segments causing the bandwidth calculation to be higher than expected

**Reply by psychopat81** [reply]
I disabled all detection and also all camera except one at the time and no difference. All my recordings are getting wiped every 10 mins or so.

**Reply by NickM-27** [reply]
you can enable debug logs for `frigate.storage` which will tell what the problematic camera is. 

In any case, I've put up a fix to at least ensure other camera recordings are not affected https://github.com/blakeblackshear/frigate/pull/15473

**Reply by psychopat81** [reply]
thanks! is there an easy way to apply your changes to my installation?

**Reply by NickM-27** [reply]
You'd need to run the latest dev image

**Reply by psychopat81** [reply]
thanks Nick, what is the tag to use in order for the docker to use the latest dev image?
Currently im on the stable branch: 'ghcr.io/blakeblackshear/frigate:stable'

I tried, dev, beta, nightly, latest none of them work.

**Reply by NickM-27** [reply]
You need to look on the repo packages section for the latest tag

**Reply by psychopat81** [reply]
latest tag is [v0.15.0-beta4](https://github.com/blakeblackshear/frigate/releases/tag/v0.15.0-beta4)

I tried that: 
ghcr.io/blakeblackshear/frigate:v0.15.0-beta4

still not working :(

**Reply by psychopat81** [reply]
No worries, found it
ghcr.io/blakeblackshear/frigate:0.15.0-beta4  :) now its working ok. thanks!
